package.json

```diff
@@ -92,7 +84,6 @@
         "supertest": "^7.0.0",
         "ts-jest": "^29.2.5",
         "ts-node-dev": "^2.0.0",
-        "tsup": "^8.2.4",
```

Comment:
tsup is still being used for the build script (and I have no plan to change this)

Reply:
ah right you are. i missed the parallel tsup build in the script. fixed in [9655efe](https://github.com/passageidentity/passage-node/pull/182/commits/9655efef48713c422f0acf36674db74f27c255ef)